test/application.test.js

```diff
@@ -128,21 +128,25 @@ describe('Feathers application', function () {
     var app = feathers()
       .configure(feathers.rest())
       .use('/todo', function (req, res, next) {
-        req.feathers.stuff = 'custom middleware';
+        req.feathers.preService = 'pre-service middleware';
         next();
-      }, todoService)
+      }, todoService, function (req, res, next) {
+        res.set('post-service', res.data.id);
```

Comment:
What I mean is to instead set something like:

```js
res.data.postService = true;
```

and then in the test check for

```js
assert.ok(data.postService, 'Post-service middleware updated response');
```

instead of `assert.equal(response.headers['post-service'], 'dishes', 'Post-service middleware updated response');`.